### PR TITLE
Upgrade bundle version to rhpam-kogito-operator.v7.11.0-1

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Many thanks for submitting your Pull Request :heart:! 

Jira Issue : https://projects.engineering.redhat.com/browse/SPMM-6427

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

File: config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -79,4 +79,5 @@ spec:
   maturity: beta
   provider:
     name: Red Hat
+  replaces: rhpam-kogito-operator.v7.11.0
   version: 7.11.0
```
Comment (AntStephenson) [inline config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml:83]:
```suggestion
  version: 7.11.0-1
```
Should this also be updated?

**Review by AntStephenson — CHANGES_REQUESTED** [review]

**Review by spolti — COMMENTED** [review]
approving, can be merged once Ant's suggestion is applied.

**Review by spolti — APPROVED** [review]

**Review by sutaakar — APPROVED** [review]

**Review by radtriste — APPROVED** [review]

File: config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (vaibhavjainwiz) [reply config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml:83]: `config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml` file is just used as a template to create actual bundle file. Version field (`version: 7.11.0`) of this file will be overriden when `make bundle` cmd run. Value of this field is controlled through make file.

Comment (AntStephenson) [reply config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml:83]: OK, no problem.

**Review by AntStephenson — APPROVED** [review]

**Pull request merged**